monero-outreach-docs/en/quickfacts/quickfacts.md

```diff
@@ -97,11 +98,9 @@ _With “tail emission” of 0.6 XMR/block, by 2040 there will be an equal amoun
 
 Although Monero is already available and being used across the globe, the community of developers have exciting goals to continue enhancing the privacy, security, and usability features of Monero and cryptocurrency in general. These are a few that are coming soon:
 
-**Bulletproofs:** This [development](https://getmonero.org/es/2017/12/07/Monero-Compatible-Bulletproofs.html) will store the blockchain efficiently and will verify transactions faster.
-
 **Kovri:** This is a major privacy upgrade, adding a fourth layer of privacy to Monero transactions. Kovri uses **[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (think of Tor on steroids) to mask the IP addresses of senders and receivers using the network. Every layer of a Monero transaction, from the sender, receiver, transaction amount, and (with Kovri) the backbone of the transaction itself, will all be private and secure.
 
-**Hardware Wallets:** Trezor Model T hardware wallet added Monero to their **[supported coins](https://trezor.io/coins/#XMR)**. **[Ledger hardware wallet](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** already available through official Monero GUI wallet. The Monero community funded a team that is building **[Kastelo, an open source hardware wallet](http://kastelo.org/)**, built on the Monero ethos. Both of these hardware wallet options have full Monero support. 
+**Hardware Wallets:** The popular Ledger and Trezor hardware wallets are working on adding Monero. (Ledger is supported since CLI 0.12.2 and GUI 0.12.3) The Monero community is funding a team that is **[building a hardware wallet](http://kastelo.org/)** built on the Monero ethos. Both of these hardware wallet options are anticipated to be operational by the end of 2018.
```

Comment:
`both of these`: but we are counting three.  
`anticipated at the end of 2018` but we are in 2019.

Reply:
Yeah I think it needs some reorganized structure, but it refers to Trezor and Ledger.

Reply:
Did the changed on this, let me know what you think :).

Reply:
I can't see the change atm. Did you pushed it?

Reply:
Yes in a different commit, let me squash so you are able to see the changes.